packages/p2p/src/components/orders/order-table-row.jsx

```diff
@@ -62,51 +70,150 @@ const OrderRowComponent = observer(({ data: order, style }) => {
-                    'orders__table-grid--active': general_store.is_active_tab,
-                    'orders__table-row--attention': !isOrderSeen(id),
-                })}
-            >
-                <Table.Cell>
-                    {(is_buy_order && !is_my_ad) || (is_sell_order && is_my_ad) ? (
-                        <Localize i18n_default_text='Buy' />
-                    ) : (
-                        <Localize i18n_default_text='Sell' />
-                    )}
-                </Table.Cell>
-                <Table.Cell>{id}</Table.Cell>
-                <Table.Cell>{other_user_details.name}</Table.Cell>
-                <Table.Cell>
-                    <div
-                        className={classNames('orders__table-status', {
-                            'orders__table-status--danger': should_highlight_danger,
-                            'orders__table-status--alert': should_highlight_alert,
-                            'orders__table-status--success': should_highlight_success,
-                            'orders__table-status--disabled': should_highlight_disabled,
+        <React.Fragment>
+            {isMobile() ? (
+                <div
+                    style={style}
+                    className={classNames('orders__expander', {
+                        'orders__expander-heighlight-danger': should_highlight_danger,
+                    })}
+                >
+                    <div className='orders__expander-top' onClick={() => order_store.setQueryDetails(order)}>
+                        <Table.Row className='orders__expander-header'>
+                            <Table.Cell
+                                className={classNames('orders__table-row orders__expander-header', {
+                                    'orders__table-grid--active': is_active,
+                                })}
+                            >
+                                <div
+                                    className={classNames('orders__expander-status', {
+                                        'orders__table-status--danger': should_highlight_danger,
+                                        'orders__table-status--alert': should_highlight_alert,
+                                        'orders__table-status--success': should_highlight_success,
+                                        'orders__table-status--disabled': should_highlight_disabled,
+                                    })}
+                                >
+                                    {status_string}
+                                </div>
+                            </Table.Cell>
+                            <Table.Cell className='orders__expander-header-right'>
+                                {is_timer_visible && <div className='orders__expander-time'>{remaining_time}</div>}
+                                <div className='orders__expander-chat'>
+                                    <Icon icon='IcChat' size={16} />
```

Comment:
This leads directly to chat, right? Where is that mentioned?

Reply:
Yes,  I've updated the code but as chat wait for the server response it takes around 1-2 seconds to show the chat component